src/sensor/ping360.h

```diff
@@ -157,7 +157,15 @@ class Ping360 : public PingSensor
     void set_range(uint newRange)
     {
         if(newRange != range()) {
+            _num_points = _firmwareMaxNumberOfPoints;
```

Comment:
so it's not possible to control range and points individually ? That's not a good thing since we are not allowing our users to run minimal distance with minimal number of points to increase the fps.

Reply:
> so it's not possible to control range and points individually ?

It is already this way in master, this pull request does not affect that. If we require direct control over number of points, we need to open a new issue to add it independently of this pull request, and we can have the option of automatic maximizing like we have implemented in master already. Very short distance scanning <2m does not represent a significant portion of use cases, and the performance gain will be marginal (the serial time saved by changing number of points is insignificant), so this is low priority in my opinon.

I think it is more useful for our application to always use the maximum resolution to provide the user with the highest detail achievable with the hardware. Range and number of points are coupled by an equation. It makes the most sense to me to allow the user to adjust only one. This makes the application easy to use.